docs/api-reference/form-props.md

```diff
@@ -95,6 +95,7 @@ React component used to customize each field of the form. See [Custom Templates]
 ## formContext
 
 You can provide a `formContext` object to the Form, which is passed down to all fields and widgets. Useful for implementing context aware fields and widgets.
+Using `{readonlyAsDisabled: false}` will make most themes use the toolkit native functionality offer readOnly fields instead of using the HTML-disabled mechanism.
```

Comment:
I think this is a more accurate description of what it does, what do you think @mdornseif @heath-freenome  ?
```suggestion
Setting `{readonlyAsDisabled: false}` on the formContext will make the antd theme treat readOnly fields as disabled.
```